### PR TITLE
Add TRA services to the service map

### PR DESCRIPTION
**Pull request by simonwhatley**
This PR adds two TRA services to the service map:

- Check a teacher’s record
- Check the children’s barred list

### CODE REVIEW

File: app/images/service-map/service-map.png
Comment (ralph-hawkins) [inline app/images/service-map/service-map.png:1]: Is it worth cropping the bottom pixels off, give they're just empty?

File: app/service-map.md [hunk 1]
```diff
@@ -129,7 +129,15 @@ The ‘Becoming a teacher’ services are part of a broader range of digital ser
...
-        "name": "Employer Access",
+        "name": "Check a teacher’s record",
+        "href": "https://check-a-teachers-record.education.gov.uk"
+      },
+      {
+        "name": "Check the children’s barred list",
+        "href": "https://check-the-childrens-barred-list.education.gov.uk"
+      },
+      {
+        "name": "Employer Access - closing soon",
```
Comment (ralph-hawkins) [inline app/service-map.md:140]: Do we know when its closing?

**Review by ralph-hawkins — APPROVED** [review]
Looks good, a couple of minor comments. I wonder if this should live in general 'Become a teacher' section of the site?

File: app/service-map.md (hunk 1 above)
Comment (simonwhatley) [reply app/service-map.md:140]: We only know it's closing soon.

File: app/images/service-map/service-map.png
Comment (simonwhatley) [reply app/images/service-map/service-map.png:1]: I've now trimmed some pixels as suggested

**Pull request merged**